### PR TITLE
Add rename/replace behavior to upload and createFolder

### PR DESCRIPTION
**Pull request by rsf123**
Fixed the changes requested in the old pull request.  Sorry for the delay

### CODE REVIEW

File: src/MsGraph.php [hunk 1]
```diff
@@ -94,13 +94,20 @@ public function connect($id = null)
                     'token_id' => $result->id,
                     'info' => $me
                 ];
-
```
Comment (dcblogdev) [inline src/MsGraph.php:97]: this is been added in now and can be removed from your PR

File: changelog.md [hunk 2]
```diff
@@ -274,4 +274,23 @@ The in a view the previous and next links can be displayed:
 
 ### 3.1.2
 
-added support for Laravel 9
```
Comment (dcblogdev) [inline changelog.md:277]: this would now be 3.1.5

**Review by dcblogdev — CHANGES_REQUESTED** [review]

**Comment by dcblogdev** [comment]
Can you update your branch to resolve the conflicts.

File: src/MsGraph.php (hunk 1 above)
Comment (rsf123) [reply src/MsGraph.php:97]: File should be the same now

File: changelog.md (hunk 2 above)
Comment (rsf123) [reply changelog.md:277]: Updated

**Review by dcblogdev — APPROVED** [review]

**Comment by dcblogdev** [comment]
just the conflicts to correct then I can merge.

**Comment by rsf123** [comment]
Should be resolved..

**Pull request merged**